spec/02-integration/03-admin_api/09-targets_routes_spec.lua

```diff
@@ -320,4 +320,52 @@ describe("Admin API", function()
       end)
     end)
   end)
+
+  describe("/upstreams/{upstream}/targets/{target}", function()
+    describe("DELETE", function()
+      local target
+
+      before_each(function()
+        assert(helpers.dao.targets:insert {
```

Comment:
But it doesn't :)

My understanding of `before_each` means that it runs before each nested test, meaning that we recreate the upstream and shut down kong after every test. And actually, the anti-pattern that I used in the previous PR to create `upstream3` is unnecessary as well.

Nevertheless, I'll add it.